cli/ballerina-cli/src/test/resources/test-resources/validProjectWithCustomMavenRepo/Ballerina.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+org = "foo"
+name = "winery"
+version = "0.1.0"
+
+[[platform.java11.dependency]]
+groupId="org.ballerinalang"
```

Comment:
If the same package is in two different repos do we need to flag from which one to resolve ?

Reply:
Usually we do not flag in maven repositories as it stops when resolved from one repository. In that case user has to keep the order of repositories

Reply:
Ok, shall we improve the test case to have dependency rom maven central as well/

Reply:
Currently Maven central repo scenarios are covered in Bindgen scenario tests. Maven Central Repo and local .m2 is added as a default repo where user does not need to specify in the toml.
https://github.com/ballerina-platform/ballerina-lang/blob/master/misc/ballerina-bindgen/src/test/resources/mvn-test-resources/balProject/Ballerina.toml